wagtail/images/templates/wagtailimages/chooser/chooser.js

```diff
@@ -109,6 +109,24 @@ function(modal) {
         return false;
     });
 
+    function popuplateTitle(context) {
+        // Note: There are two inputs with `#id_title` on the page.
+        // The page title and image title. Select the input inside the model body.
+        var fileWidget = $('#id_file', context);
+        fileWidget.on('change', function () {
+            var titleWidget = $('#id_title', context);
+            var title = titleWidget.val();
+            if (title === '') {
+                // The file widget value example: `C:\fakepath\image.jpg`
+                var parts = fileWidget.val().split('\\');
```

Comment:
Will this handle UNIX paths as well? It doesn’t seem so.

Reply:
@BertrandBordage I'm on unix. OSX is certified unix right? It is a weird path. What value does http://jsfiddle.net/QPUJa/ give you when you select a file? If it is something else maybe I can create a better js.

Reply:
Yeah, OSX is based on UNIX standards.
I also get a weird `C:\fakepath\filename`. I’m not sure whether it’s coming from jQuery or directly JavaScript, but that’s clearly misleading. You don’t have to do anything about it, then :)

Reply:
`C:\fakepath` is actually a quirk required by the HTML specification: https://html.spec.whatwg.org/multipage/input.html#file-upload-state-(type=file)

Reply:
Do I need to do something to get this merged? Just let me know ;)

Reply:
About the path, do nothing, but about `populateTitle` above, there is still one commit missing :)

Reply:
Mmm, I didn't get the Github UI, I thought zerolab fixed the typo. :D